### ISSUE TITLE
Inspect Editor Tokens - hover stays open

### ISSUE THREAD

**Issue by aeschli** [microsoft/monaco-editor#1787]
Testing https://github.com/microsoft/vscode/issues/89392

Firefox, Shadow Dom

- Run the `Inspect Editor Token` command
- I wasn't able to hide the hover, Escape or setting focus to an element outside the editor didn't work

**Comment by alexdima** [comment]
This is unrelate to shadow dom, we simply don't have a way to close that at this time.

**Comment by nrayburn-tech** [comment]
@alexdima How exactly would we want to close the hover?  Personally, I only want it closed when it gets in the way of the text around what I am looking at.  So I would only want it hidden until I click onto the next token.  What are your thoughts?
Here is how I have it modified currently.  When pressing escape, I set `visibility = hidden`.  On the widget compute, I set the `visibility = inherit`.  (I could also just add another listener, but wasn't really sure if that was necessary.)
![CloseHover](https://user-images.githubusercontent.com/52075362/74574220-8e306f80-4f48-11ea-8d2e-2598f0ec38d0.gif)
